### PR TITLE
remove map filters, use data

### PR DESCRIPTION
**Pull request by MaciejReimann**
Quick fox to make the filters work with the map.

### CODE REVIEW

**Review by jciepka — APPROVED** [review]

File: src/pages/requests/index.tsx [hunk 1]
```diff
@@ -129,16 +129,6 @@ export function Requests() {
...
 
-  // TODO: move this logic to map component - it should get filters via context and process them accordingly
-  // TODO: fix filter mapping - it should use category_id
-  // TODO: add some comments / typing explaining what kind of black the magic is happening here :)
-  const layerFilterCategory = activeFilterItems.length
-    ? ["in", ["get", "category"], ["array", ["literal", activeFilterItems]]]
-    : ["==", ["get", "category"], "ALL"];
-
-  const layerFilterDate = activeDateFilter ? ["==", ["get", "date"], ["string", activeDateFilter]] : ["boolean", true];
-  const layerFilter = ["all", layerFilterCategory, layerFilterDate];
```
Comment (AntonRaichuk) [inline src/pages/requests/index.tsx:140]: i'm confused, this totally breaks current filter logic, why did you send this change?

**Review by AntonRaichuk — CHANGES_REQUESTED** [review]
This change currently breaks the map, no?

File: src/pages/requests/index.tsx (hunk 1 above)
Comment (MaciejReimann) [reply src/pages/requests/index.tsx:140]: This actually fixes it. This is what I tried to explain when we're talking. We don't need that, we need proper DATA. :)
No more, no less.

Comment (MaciejReimann) [reply src/pages/requests/index.tsx:140]: We filter data BEFORE it enters the map module. We don't need to filter it AGAIN.

Comment (MaciejReimann) [reply src/pages/requests/index.tsx:140]: But I understand your confusion, I was confused too!

Comment (AntonRaichuk) [reply src/pages/requests/index.tsx:140]: ah, wtf, geojson is filtered data.. ok, let me see this working with my own eyes before merging

**Review by AntonRaichuk — APPROVED** [review]
somehow 'git checkout feature/filters-for-map' fails for me, so i'll test in dev

**Comment by AntonRaichuk** [comment]
@MaciejReimann you are totally right! it works.

i must have made a mistake when playing with regions.
tomorrow will simplify my PR and remove the useEffect alltogether, thanks finding this new solution

**Comment by AntonRaichuk** [comment]
Aha, I didn't actually pay attention that you made the switch from using all the data to just the filtered version as part of this PR - https://github.com/ukraine-taskforce/requests-dashboard/pull/52/files

**Pull request merged**